src/test/java/io/appium/java_client/android/AndroidDriverTest.java

```diff
@@ -209,8 +209,9 @@
     }
 
     @Test public void pullFileTest() {
+        //Works only on rooted devices/Emulator and adb is running as root
```

Comment:
this actually depends on the actual path. For example the content of `/sdcard` is available without root access